fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/api/registry/SpreadableBlockRegistry.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.registry;
+
+import com.google.common.collect.ImmutableMap;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Nullable;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.fabric.api.util.Block2ObjectMap;
+import net.fabricmc.fabric.impl.content.registry.SpreadableBlockRegistryImpl;
+
+/**
+ * Registries of Blocks to BlockStates, defining the spreadable BlockState to replace
+ * a bare Block with when a particular type of spreadable Block spreads to the bare black.
+ *
+ * <p>For example, to get the registry for Mycelium block spread and then register
+ * a modded block pair where Mycelium spreading to MY_DIRT generates MY_MYCELIUM:
+ *
+ * <pre>{@code
+ * SpreadableBlockRegistry.getOrCreateInstance(SpreadableBlockRegistry.MYCELIUM)
+ *        .add(MyModBlocks.MY_DIRT, MyModBlocks.MY_MYCELIUM.getDefaultState());
+ * }</pre>
+ */
+@ApiStatus.NonExtendable
+public interface SpreadableBlockRegistry extends Block2ObjectMap<BlockState> {
+	/**
+	 * Registry ID for Minecraft Grass Block type spreadable blocks.
```

Comment:
```suggestion
	 * Registry ID for grass-type spreadable blocks.
```
No need to say "Minecraft" here, and can be shorter in general